### PR TITLE
Desk version

### PR DESCRIPTION
**Pull request by joel-kalema**
- [ ] I need to make sure that my desktop version looks like the one in the Figma design template that I chose.
- [ ] IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version 
       and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please 
       ignore this part of the design template and use only one input for the full name for all versions.
- [ ] Use media queries in my CSS file to make my desktop version look like the template selected (let's assume that the 
       desktop width should be in the 992px - 1100px range).
- [ ] In order to layout the elements on the page I should use Grid and Flexbox.
- [ ] You don't need to implement any functionality that requires JavaScript.
- [ ] I must replicate the design as much as possible (e.g., font, colors, images, tests, margins) using the info from the Figma 
      templates

### CODE REVIEW

File: media.css [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+  }
+
+  .contact .text {
+    margin: 50px auto;
+    width: 320px;
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 900px) and (min-width: 1100px) {
```
Comment (codepantha) [inline media.css]: - [ ] You have possibly erroneously specified `min-width` twice. Please ensure that your desktop `min-width` is between `992px `and `1100px` 👍

File: index.html [hunk 2]
```diff
@@ -70,53 +81,65 @@ <h5>Skills</h5>
...
+            </p>
+            <div class="langage-used">
+              <ul>
+                <li>Css</li>
+                <li class="colum1">Html</li>
+                <li class="colum2">Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <button type="button">See Project</button>
```
Comment (codepantha) [inline index.html:130]: - [ ] Please let's ensure to stick to the colors and design specified in the Figma design. In this case, the `background-color` of the buttons don't match the specifications. 🙏 Same goes for all the "see project" buttons.

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @joel-kalema 👋, 

Terrific job so far! 👍 . However, there are still a few changes you need to make in order to get approval but you're really close!

## To Highlight ⚡ 
- Descriptive readme ✔️ 
- Beautiful UI ✔️ 
- Meaningful PR ✔️ 


## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AnselemOdims — APPROVED** [review]
### Hi  @joel-kalema 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**